### PR TITLE
Tidy up `notification.md`

### PR DESCRIPTION
**Pull request by ciaranevans**
This PR tidies up `notification.md` by removing command prefixes and formatting YAML blocks

It also renames the navigation entry from `Notify` to `Notifications`

Signed-off-by: Ciaran Evans <ciaran@reliably.com>

### CODE REVIEW

File: sources/reference/usage/notification.md [hunk 1]
```diff
@@ -220,9 +215,8 @@ amend it, please open an [issue][slissue] there.
 
 The Chaos Toolkit does its best to not break the experiment when an event
 could not be pushed. If you do not see the notification you were expecting,
-you should start investigating in the `chaostoolkit.og` file. Indeed, in that
-case the error is logged at the `DEBUG` level with, hopefully, enough
-information why the event could not be sent.
+you should start investigating in the `chaostoolkit.log` file. If Chaos Toolkit could handle the issue
```
Comment (Lawouach) [inline sources/reference/usage/notification.md:218]: Keep line lengths to under 80 ;)

File: mkdocs.yml [hunk 2]
```diff
@@ -81,7 +81,7 @@ nav:
         - Init: reference/usage/init.md
         - Run: reference/usage/run.md
         - Report: reference/usage/report.md
-        - Notify: reference/usage/notification.md
+        - Notifications: reference/usage/notification.md
```
Comment (Lawouach) [inline mkdocs.yml:84]: Interestingly, we chopse Notify for its action driven aspect. They are all verbs, not nouns.

Comment (ciaranevans) [reply mkdocs.yml:84]: I'd be inclined to agree, if it were an actual command you can run, but it isn't

Chaos Toolkit can do notifications, but you can't do `chaos notify`

Comment (Lawouach) [reply mkdocs.yml:84]: Mh, that makes sense indeed. Actually, that whole notifications mechanic is obsleted by controls anyway. But controls we haven't shipped yet. Okay make the change for now.

**Comment by ciaranevans** [comment]
@Lawouach Can you approve this please? 😄

**Review by Lawouach — APPROVED** [review]

**Pull request merged**